### ISSUE TITLE
Bower support

### ISSUE THREAD

**Issue by Soviut** [devbridge/html5-file-uploader#10]
It would be really nice if this library had a bower.json file so that they could be installed as a dependency via `bower install html5-file-uploader`.

**Comment by thomaswelton** [comment]
Yeah I agree. Downloading via their websites prompts for you email address. And the layout of this repo isn't that great. The actual source is only in a demo directory.
Anyway I've gone ahead and registered this on the bower registry `bower install html5-file-uploader` will work.

But if you maintainers of this repo can "tag" a release using a server version number that'd be great and improve compatibility and reliability.
